### ISSUE TITLE
[Question] What is the recommended API for deprecated SetDynamicRange?

### ISSUE THREAD

**Issue by Benjamin-Tan** [NVIDIA/TensorRT#4292]
From the sample code https://github.com/NVIDIA/TensorRT/blob/release/10.7/samples/sampleINT8API/sampleINT8API.cpp#L338, it is still using the deprecated API `setDynamicRange` since TensorRT 10.1. What is the recommended way to implement this? Is there any example on explicit quantization in C++?

**Comment by lix19937** [comment]
Yes, it deprecated in TensorRT 10.1. Superseded by explicit quantization. Explicit quantization means QAT, see https://github.com/NVIDIA/TensorRT/tree/release/10.7/tools/pytorch-quantization .
